docs/data/data-grid/row-ordering/row-ordering.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Data Grid - Row ordering
+---
+
+# Data grid - Row ordering [<span class="plan-pro"></span>](https://mui.com/store/items/mui-x-pro/)
+
+<p class="description">Drag and drop your rows to reorder them.</p>
+
+Row reordering allows to rearrange rows by dragging the special reordering cell.
+
+By default, row reordering is disabled.
```

Comment:
I would add a paragraph describing the default behavior with a link to the Sorting page to let people know how those two features interplay.

I think this information is worth displaying up-front instead of in a warning at the end of the page